### ISSUE TITLE
Can't unset files as default

### ISSUE THREAD

**Issue by ranjeetsharma0** [files-community/Files#6824]
### Description

I can't unset files as default and make windows explorer default again. And yes, when it asks for administration permission in more information it shows location of UnsetFilesAsDefault.reg. but there is no such .reg file in that directory.

### Steps To Reproduce

1.Settings
2. Experimental
3. Set files as default
4. restart pc
5. unset files as default(this doesn't do what it's supposed to)

### Expected behavior

It should unset itself as default and make windows explorer default again.

### Files Version

Version: 2.0.34.0 OS Version: 10.0.19044.1320

### Windows Version

Edition	Windows 10 Pro Version	21H2 Installed on	‎03-‎07-‎2021 OS build	19044.1320 Experience	Windows Feature Experience Pack 120.2212.3920.0

### Relevant Assets

[debug_fulltrust.log](https://github.com/files-community/Files/files/7502052/debug_fulltrust.log)
[debug.log](https://github.com/files-community/Files/files/7502053/debug.log)

**Comment by yaichenbaum** [comment]
@ranjeetsharma0 did you reinstall Files?

**Comment by fuxinsen** [comment]
I also find this bug. I reinstall Files 3 times but can‘t set explorer as default.

**Comment by fuxinsen** [comment]
It's easy to replay this bug, turn on the experimental function **set files to default** and then cancel.  Now the default explorer cannot be opened include WIN + E.

**Comment by yaichenbaum** [comment]
> I also find this bug. I reinstall Files 3 times but can‘t set explorer as default.

This issue is caused by reinstalling Files. Anyways, we'll have a fix in the next update.

**Comment by jardelvdm** [comment]
When I tried to uncheck the `Set Files as default` and did not work, I took a picture of the path the registry would act  in the file `UnsetFilesAsDefault.reg` and then noticed that the directory was empty.

So I found and did follow the step 2 of [these instructions](https://files.community/docs/configuring/replace-file-explorer) and extracted the `.reg` files into the directory. After that the `unset files as default` works nicely.
